subscriptions/providers/apple_in_app/__init__.py

```diff
@@ -102,61 +103,83 @@ def check_payments(self, payments: Iterable[SubscriptionPayment]):
                 raise AppleSubscriptionNotCompletedError(payment.provider_transaction_id)
 
     @classmethod
-    def _get_validated_in_app_product(cls, response: AppleVerifyReceiptResponse) -> AppleInApp:
+    def _is_receipt_valid(cls, response: AppleVerifyReceiptResponse) -> None:
```

Comment:
"is_..." should return `bool`. I'd call it `_check_receipt_is_valid` or `_raise_if_invalid` or something like that.